test/e2e/network/kube_proxy.go

```diff
@@ -77,6 +77,16 @@ var _ = SIGDescribe("Network", func() {
 
 		zero := int64(0)
 
+		// Some distributions (Ubuntu 16.04 etc.) don't support the proc file.
+		_, err := framework.IssueSSHCommandWithResult(
+			"sudo ls /proc/net/nf_conntrack",
```

Comment:
do you also want to consider where the machine doesn't have `sudo` correctly configured, or not configured with promptless?

Reply:
I see your point, we might be able to check it without `sudo`.
but I hesitated to do it because the check of `/proc/net/nf_conntrack` is using `sudo` and there are 77 places of `sudo` under test/e2e like
```
~/go/src/k8s.io/kubernetes/test/e2e$ grep "sudo " * -R | wc
     77     841    8443
```
In addition, I am not sure non-root users always can check the file.

Reply:
do you need `sudo`. why not just `ls`?

Reply:
OK, will remove it.